src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -443,5 +485,62 @@ private static byte GetSeverityNumber(LogLevel logLevel)
                 return 1;
         }
     }
+
+    private static int SanitizeCategoryName(byte[] buffer, int cursor, ref int validNameLength, string categoryName)
```

Comment:
Add the expected behavior of this method from the issue #289 as comments before this method.